portfolio/src/main/webapp/about.html

```diff
@@ -1,20 +1,53 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>About</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <ul id="nav">
-        <li><a href="index.html">Home</a></li>
-        <li><a href="about.html">About</a></li>
-        <li><a href="resume.html">Resume</a></li>
-        <li><a href="contact.html">Contact</a></li>
-    </ul>
-    <div id="content">
-      <p>My name is Coco Sack. I am a current Sophomore at Yale University, currently studying Computer Science and Psychology. I love to cook, travel, read.</p>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>About</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body id = about>
+        <div id="frametop" class="frame"></div>
+        <div id="framebottom" class="frame"></div>
+        <div id="frameright" class="frame"></div>
+        <div id="frameleft" class="frame"></div>
+        <ul id="nav">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="resume.html">Resume</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+        <div id="content">
+            <h2>
+                <span
+                class="txt-rotate"
+                data-period="10000"
+                data-rotate='["About" ]'></span>
+            </h2>
+            <div id="aboutContainer">
+                <h3>About Me</h3>
+                <div class = rectImage>
+                    <img class="aboutImg" src = "/images/Portrait2.jpg" alt="About Me Photo">
```

Comment:
about-img: Separate words in ID and class names by a hyphen here and below

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters